webapp/app/data_access/storage/session_storage.py

```diff
@@ -0,0 +1,45 @@
+from typing import Optional
+
+from pydantic import MissingError
+from flask_login import current_user
+
+from app.data_access.form_data_controller import FormDataController
+from app.data_access.storage.form_storage import FormStorage
+
+
+class SessionStorage(FormStorage):
```

Comment:
We could maybe make both SessionStorage and CookieStorage be of singleton use so that everytime we need them they are not newly instantiated? wdyt?

Reply:
Not necessary :) why should we need a singleton?

Reply:
I like singleton :) but if you think is not needed, then it is ok how it is implemented